src/vaults/AsyncRequests.sol

```diff
@@ -128,7 +128,7 @@ contract AsyncRequests is BaseInvestmentManager, IAsyncRequests {
         require(state.pendingCancelDepositRequest != true, CancellationIsPending());
 
         state.pendingDepositRequest += assets;
-        IPoolEscrow(poolEscrowProvider.escrow(poolId)).pendingDepositIncrease(
+        IPoolEscrow(address(poolEscrowProvider.escrow(poolId))).pendingDepositIncrease(
```

Comment:
NIT. THis can be:
```suggestion
        poolEscrowProvider.escrow(poolId).pendingDepositIncrease(
```
If poolEscrowProvider already store IPoolEscrow in the mapping